src/server/api/creation.ts

```diff
@@ -28,13 +29,33 @@ export const creationRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityP
   }, async (request, reply) => {
     const { actor } = request.params
 
+    const existedAlready = (await store.actorsDb.keys().all()).some(k => k === actor)
```

Comment:
Instead could you add a `actorsDB.has(actor)` API? This seems kinda inefficient.

Reply:
i tried that, but at least my editor tells me that `.has` doesn't exist in abstract-level. `.get` returns `any` ( :( ), so i'm not really sure how to do this "properly"

Reply:
Ah, you can perform a `.get` and check that the value is not undefined. https://www.npmjs.com/package/abstract-level#dbgetkey-options

Or I think actorsDb.get throws an error on not found so maybe .has can catch that and return false